Services/AzineWebViewService.php

```diff
@@ -31,11 +33,10 @@ public function getTemplatesForWebPreView()
      */
     public function getTestMailAccounts()
     {
-        $emails = array();
         // override this method to add your own emails
-        // $emails = $this->addTestMailAccount($emails, 'Testmail-account for MS Outlook',	'your.email.address@for.an.outlook.client.com');
-        // $emails = $this->addTestMailAccount($emails, 'Testmail-account for Gmail', 	'your.email.address@gmail');
-        return $emails;
+        // $this->addTestMailAccount('Testmail-account for MS Outlook',	'your.email.address@for.an.outlook.client.com');
+        // $this->addTestMailAccount('Testmail-account for Gmail', 	'your.email.address@gmail');
```

Comment:
Carefull here,you change the behaviour, calling thé get....() Will add new templates to thé service. I am not sure this os thé expected  behaviour.